packages/eslint-config-react-native/src/rules/import.ts

```diff
@@ -0,0 +1,99 @@
+import { ESLint } from 'eslint';
+
+const Config: ESLint.ConfigData = {
+  plugins: ['import'],
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: [
+          '.js',
+          '.jsx',
+          '.ts',
+          '.tsx',
+          '.d.ts',
+          '.android.js',
+          '.android.jsx',
+          '.android.ts',
+          '.android.tsx',
+          '.ios.js',
+          '.ios.jsx',
+          '.ios.ts',
+          '.ios.tsx',
+        ],
+      },
+    },
+  },
+  rules: {
+    // Forbid the use of extraneous packages
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: [
+          '**/__tests__/**/*',
+          'e2e/**/*',
+          '**/*.test.{js,ts,tsx}',
+          '**/*.doc.{js,ts,tsx}',
+          'jest/**/*',
+          'scripts/**/*',
+          'app/testing/**/*',
+        ],
+        peerDependencies: true,
+        optionalDependencies: true,
+      },
+    ],
+    // Ensure imports point to a file/module that can be resolved
+    'import/no-unresolved': [
+      'error',
+      {
+        ignore: ['~/'], // @todo remove it to have better file unresolved detecion
```

Comment:
Should we create a ticket for this?

Reply:
I don't know 🤷🏼‍♂️ I didn't really touch to the initial config. It was already there. Do you have in mind the context of this comment addition?